internal/store/serverless_private_endpoints.go

```diff
@@ -14,6 +14,8 @@
 
 package store
 
+// TODO: remove?, moved all use to atlas/ ?
```

Comment:
please link TODOs to a CLOUDP

Reply:
These turned to be leftovers, make no sense so I am removing them on next update.

Reply:
Done

Reply:
some of these TODOs are still in the changes

Reply:
I need to check, because they are removed in my local. Not sure what happened.